files/src/utils/test-helpers/test-helper.ts

```diff
@@ -0,0 +1,7 @@
+import { setApp, start } from '@glimmer/test-helpers';
+import App from '../../main';
+
+QUnit.config.autostart = false;
```

Comment:
QUnit gets defined as an [AMD module](https://github.com/qunitjs/qunit/blob/35d960bb2c2b52662264f9c7698ee38f7af41df6/src/export.js#L31-L33), but it's anonymous. Maybe we could leverage [this](https://ember-cli.com/user-guide/#standard-anonymous-amd-asset)?